### ISSUE TITLE
problems with config

### ISSUE THREAD

**Issue by h4ck3rm1k3** [pavlobaron/wpcorpus#2]
hi there, 
my change to config did not work perfectly, my latest version does not use config for everything. https://github.com/h4ck3rm1k3/wpcorpus/blob/master/lib/wpcorpus/process.py#L77

But in the good news:
In any case I have gotten the training running finally for a single dumped file.

**Comment by pavlobaron** [comment]
yeah, I don't like this half-hardcoded part, need to rework it some day.
